packages/webpack-cli/package.json

```diff
@@ -13,7 +13,7 @@
   },
   "main": "./lib/index.js",
   "engines": {
-    "node": ">=10.13.0"
+    "node": ">=12.13.0"
```

Comment:
@jamesgeorge007 Let's add `engines.node` to all our packages to avoid future problems :+1: